src/test/java/org/graylog/integrations/notifications/types/microsoftTeams/TeamsEventNotificationConfigTest.java

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+package org.graylog.integrations.notifications.types.microsoftTeams;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import org.graylog.events.event.EventDto;
+import org.graylog.events.notifications.EventNotificationExecutionJob;
+import org.graylog2.contentpacks.EntityDescriptorIds;
+import org.graylog2.plugin.rest.ValidationResult;
+import org.joda.time.DateTime;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertEquals;
+
+
+public class TeamsEventNotificationConfigTest {
+
+        @Test
+        public void validate_succeeds_whenWebhookUrlIsValidUrl() {
+            TeamsEventNotificationConfig teamsEventNotificationConfig = TeamsEventNotificationConfig.builder()
+                    .webhookUrl("http://graylog.org")
+                    .build();
+            ValidationResult result = teamsEventNotificationConfig.validate();
+            Map errors = result.getErrors();
+            assertThat(errors).size().isEqualTo(0);
+        }
+
+        @Test
+        public void validate_succeeds_whenWebhookUrlIsValidTeamsUrl() {
+            TeamsEventNotificationConfig teamsEventNotificationConfig = TeamsEventNotificationConfig.builder()
+                    .webhookUrl("https://teams.webhook.office.com/webhookb2/d6068ba8-584c-41--bb7e-3b112e9b1ff4/IncomingWebhook/440/846c")
+                    .build();
+            ValidationResult result = teamsEventNotificationConfig.validate();
+            Map errors = result.getErrors();
+            assertThat(errors).size().isEqualTo(0);
```

Comment:
The same can be accomplished with assertEquals, which is a dependency we already have, and reduces fanout complexity.

Reply:
Replaced assertThat() with assertEquals()